noir-projects/aztec-nr/aztec/src/test/helpers/test_environment.nr

```diff
@@ -14,6 +14,11 @@ use crate::oracle::{
 use protocol_types::constants::PUBLIC_DISPATCH_SELECTOR;
 use protocol_types::traits::Packable;
 
+// This is the first nullifier emitted from the TXe. It is an arbitrary number assigned in the TXe that is
+// larger than the first prefilled subtree of the nullifier tree. We use know FIRST_NULLIFIER_EMITTED_IN_TXE exists
```

Comment:
```suggestion
// larger than the first prefilled subtree of the nullifier tree. We know FIRST_NULLIFIER_EMITTED_IN_TXE exists
```
Where does TXE emit this? I cannot find that the global variable used when emitting it. Would use it for that if possible.

Reply:
TBH I find this quite problematic as this breaks the tests if that random value injected in txe_oracle.ts changes. I think the best thing here would be to just not use it and insert our own nullifier in the tests. That shouldn't be a problem, no?